script/Arcueid/Day 7/07_00_ARC07_3E.txt

```diff
@@ -1,7 +1,7 @@
 [sha:1243f5e061aee1adbf432f8c5d5294628588f5d5]{
 -- Page 10012, Offset 17239.
 -- アルクェイドは夜の街を歩いていく。
-Arcueid walks through the night city.
+Arcueid walks through the city night.
```

Comment:
Heads up that I am in the middle of editing this file, so a fair bit is gonna change. Not going to review it here given that.